app/src/main/java/com/projectit3048c/service/FoodService.kt

```diff
@@ -12,10 +12,10 @@ import retrofit2.awaitResponse
 interface IFoodService {
     suspend fun fetchFoods() : List<Food>?
 }
-class FoodService : IFoodService {
+internal class FoodService : IFoodService {
     override suspend fun fetchFoods() : List<Food>? {
         return withContext(Dispatchers.IO) {
-            val service = RetrofitClientInstance.retrofitInstance?.create(iFoodDAO::class.java)
+            val service = RetrofitClientInstance.retrofitInstance?.create(IFoodDAO::class.java)
```

Comment:
Within this commit, I changed the name of the dao to make the naming conventions more cohesive

Reply:
Duplicates a previous review.  When that happens, first in wins.